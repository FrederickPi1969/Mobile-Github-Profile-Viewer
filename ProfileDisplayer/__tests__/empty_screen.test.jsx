import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import EmptyScreen from '../src/empty_screen';

// Note: test renderer must be required after react-native.

it('renders correctly', () => {
  const tree = renderer.create(<EmptyScreen
    navigation={null}
    route={{ params: { emptyPrompt: 'Empty!' } }}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
