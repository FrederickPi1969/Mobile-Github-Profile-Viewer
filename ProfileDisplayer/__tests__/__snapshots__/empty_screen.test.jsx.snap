// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffb8ce",
      "height": "100vh",
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#2902eb",
          "fontSize": 18,
        }
      }
    >
      &lt; Back
    </Text>
  </View>
  <Image
    source={
      Object {
        "testUri": "../../../assets/empty.png",
      }
    }
    style={
      Object {
        "height": "60vw",
        "marginLeft": "auto",
        "marginRight": "auto",
        "marginTop": "25%",
        "width": "60vw",
      }
    }
  />
  <Text
    style={
      Object {
        "fontFamily": "Comic Sans MS",
        "fontSize": 20,
        "fontWeight": "bold",
        "marginTop": "2vh",
        "textAlign": "center",
      }
    }
  >
    Empty!
  </Text>
</View>
`;
